test/built-ins/String/prototype/codePointAt/return-first-code-unit.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.3
+description: >
+  Return first code unit value of the element at index position if first >=
+  0xD800 or first <= 0xDBFF and second < 0xDC00 or second > 0xDFFF.
```

Comment:
This test is good, but it took me a little while to figure out what is going on here. A better name/description would say something surrogate pairs--maybe "Returns code point of LeadSurrogate if not followed by a valid TrailSurrogate."